dashboard/templates/base.html

```diff
@@ -46,8 +46,8 @@
     <footer>
         <hr>
         <div class="row">
-            <div class="col text-left">Copyright © 2018 The Regents of the University of Michigan</div>
-            <div class="col text-right">Last updated on {{last_updated|date:"m/d/Y P T"}}</div>
+            <div id="footer-left" class="col text-left"><span id="copyright">Copyright © 2018 The Regents of the University of Michigan</span></div>
+            <div id="footer-right" class="col text-right"><span id="updated">Data last updated on {{last_updated|date:"m/d/Y P T"}}</span><span id="version">:Version {% include "git_head_version.html" %}</span></div>
```

Comment:
Should "Version" be "GitHub Version" to be more specific?

Reply:
Well maybe "Git" version. I didn't want to change it if we moved to GitLab

Reply:
Though when/if we do that we could worry about that then ;)

Reply:
![image](https://user-images.githubusercontent.com/27447/47877364-19d4ca00-ddf2-11e8-8a3a-48b48d216753.png)

Reply:
This is good. But I feel that we need to have the Openshift Build info on the page some where invisible that would be good. That will be helpful in case of roll back of code to earlier version build/deployment. It would be a good gaze as we know that we have Build#123 or Deploy#23, git branch/tag. Much like what CCM/CPM doing it as an invisible button.

`OPENSHIFT_BUILD_NAMESPACE: ccm OPENSHIFT_BUILD_NAME: canvascoursemanager-12 GIT_SOURCE: https://github.com/tl-its-umich-edu/canvas_course_manager.git GIT_BRANCH: 4.0.1 GIT_COMMIT: fb68ce8d56741338f6c1fa414201be5c19b91023`

Reply:
May be another ticket

Reply:
Yeah, while invisible button is convenient, it's not a good solution for accessibly reasons. Maybe for the python programs we can just add this information to the Django Admin UI. That seems a good solution all-around. Another ticket.

Reply:
Yes, I think openshift variable is just for the dev to make sure the deploy/build it is what it is visually.